docs/manage-fileset-metadata-using-gravitino.md

```diff
@@ -0,0 +1,413 @@
+---
+title: "Manage fileset metadata using Gravitino"
+slug: /manage-fileset-metadata-using-gravitino
+date: 2024-4-2
+keyword: Gravitino fileset metadata manage
+license: Copyright 2024 Datastrato Pvt Ltd. This software is licensed under the Apache License version 2.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page introduces how to manage fileset metadata by Gravitino. Fileset is a concept brought
+out in Gravitino, which is a collection of files and directories. Users can leverage
+fileset to manage non-tabular data like training datasets, raw data.
+
+Typically, a fileset is mapping to a directory on a file system like HDFS, S3, ADLS, GCS, etc.
+With fileset managed by Gravitino, the non-tabular data can be managed as assets together with
+tabular data and others in Gravitino with a unified way.
+
+After fileset is created, users can easily access, manage the files/directories through
+Fileset's identifier, without needing to know the physical path of the managed datasets. Also, with
+unified access control mechanism, filesets can also be managed via the same role based access
+control mechanism without needing to set access controls to different storages.
+
+To use fileset, we assume that:
```

Comment:
Could we avoid using `we`?

Reply:
OK, let me double confirm all the places using  "we".